site/docs/master/style-guide.md

```diff
@@ -0,0 +1,313 @@
+# Velero Documentation Style Guide
+
+_This style guide is adapted from the [Kubernetes style guide](https://kubernetes.io/docs/contribute/style/style-guide/)._
+
+This page outlines writing style guidelines for the Velero documentation.
+These are guidelines, not rules. Use your best judgment as you write documentation, and feel free to propose changes to these guidelines. Changes to the style guide are made by the Velero maintainers as a group. To propose a change or addition, create an issue/PR, or add it to the [community meeting agenda](https://hackmd.io/Jq6F5zqZR7S80CeDWUklkA) and attend the meeting to participate in the discussion.
```

Comment:
our link to the community meeting agenda could change (as evident when we reached max characters last week). better to link to the community page at https://velero.io/community/.

Reply:
@michmike Jonas kept the note the same, but copied the old contents into a new note, so the link didn't actually change.

Reply:
Do we think this link is likely to change a lot? If so, it might make sense to point to the community page, so there is only one place to update the link, or create a variable that we can use where ever we need to, and just update in the config file